desktop/assets/analytics.coffee

```diff
@@ -12,8 +12,14 @@ require '../analytics/main_layout.js'
 require '../analytics/before_ready.js'
 
 $ -> analytics.ready ->
+  $.ajax '//munchkin.marketo.net/munchkin.js',
```

Comment:
I see that you're loading this script via `$.ajax`, but what you might want instead is a more traditional way to load it via a `<script>` tag. You could simply [add that here](https://github.com/artsy/force/blob/master/desktop/components/main_layout/templates/scripts.jade#L6) along-side the Segment script tag. e.g. `script( src='//munchkin.marketo.net/munchkin.js' )`